### PR TITLE
Added link to the SciPy 2019 conference talk

### PR DESCRIPTION
**Pull request by CsatiZoltan**
Closes: #88

### CODE REVIEW

File: doc/getting_started.rst [hunk 1]
```diff
@@ -506,5 +506,9 @@ This is of course a toy example. For the full dataset and analysis, see:
 But we hope this minimal example will serve for inspiration for your
 future analysis of skeleton images.
 
+If you are interested in how `numba <https://numba.pydata.org/>`_ was used
+to accelerate some parts of Skan, check out my `talk <https://www.youtube.com/watch?v=0pUPNMglnaE>`_
+on the the SciPy 2019 conference.
```
Comment (jni) [inline doc/getting_started.rst:511]:
```suggestion
If you are interested in how we used `numba <https://numba.pydata.org/>`_
to accelerate some parts of Skan, check out `jni's talk <https://www.youtube.com/watch?v=0pUPNMglnaE>`_
at the the SciPy 2019 conference.
```

**Comment by jni** [comment]
Never mind, apparently I don't need you to accept them :sweat_smile:. That's new I think...!

**Comment by CsatiZoltan** [comment]
Glad you merged. Comments on your items:
- I know that style guides advise the active voice, but I got used to the passive voice in academia.
- I hesitated between "I" and "we", but at the bottom of your Sphinx documentation, "Copyright 2017, Juan Nunez-Iglesias" is written, so I imagined myself to the place of the reader who can infer from this information your identity.
- As a non-native speaker, sometimes I have issues with the correct use of prepositions.

> apparently I don't need you to accept them

When I opened the PR, there was an option to allow the maintainer to change the PR. I ticked the box, that's why you could modify it, I guess.

**Comment by CsatiZoltan** [comment]
Unfortunately, I wrote two "the":

ksnip_20200731-081707

**Pull request merged**